src/site/stages/build/silversmith.js

```diff
@@ -41,10 +25,57 @@ module.exports = () => {
   const smith = Metalsmith(__dirname);
 
   smith.stepStats = [];
+  let stepCount = 0;
+  let consoleDirty = false;
+  let lastLogLineLength = 0;
+
+  // Spy on stdout & stderr to see if we need to write the full logStepEnd line
```

Comment:
Could be more explicit and say that we only write the full `logStepEnd` line if there's any output during the step.

Reply:
Ah, that's how people say it with words that are more sense-making. Copy that! :+1: